lib/stqueue/process.rb

```diff
@@ -127,6 +128,18 @@ def delete
       nil
     end
 
+    def increase_busy
+      busy = STQueue.store.busy(queue_name) + 1
```

Comment:
@mrlewap , @melnik0v 
When the queue is not exist yet
https://i.imgur.com/qNbaH9h.png

```
[7] pry(main)> STQueue.store.busy('stqueued_process_csv_9219')
=> nil
[8] pry(main)> STQueue.store.busy('stqueued_process_csv_9219') + 1
NoMethodError: undefined method `+' for nil:NilClass
from (pry):8:in `<main>'

```
Please fix it, thanks